code/game/machinery/doors/blast_door.dm

```diff
@@ -18,6 +18,7 @@
 	var/icon_state_opening = null
 	var/icon_state_closed = null
 	var/icon_state_closing = null
+	var/shutter = 0
```

Comment:
Wouldn't a better variable be `shutter_damage`? This would future-proof as well in case we decide to add further shutters/blast doors down the line.

Reply:
Yes, I think that would be the better way to go at it.
shutter_damage with a default set to DOOR_CRUSH_DAMAGE